### PR TITLE
Fix Custom Submit Buttons

### PR DESCRIPTION
**Pull request by csolbs24**
# Fix Custom Submit Buttons

## Description

Due to a spelling mistake in the legacy constructor `submit_butons` field didn't properly map to the `submit_buttons` so it would always use the default. 

## PR Checklist

- [x] Merged latest main
- [x] Version number in `package.json` has been bumped since last release
- [x] Version numbers match between package `package.json` and `src/version.js`
- [x] Ran `npm install` and `npm run build` AFTER bumping the version number
- [x] Updated documentation if necessary (currently just in `api_spec.md`)
- [x] Added changes to `changelog.md`

## Breaking API Changes

Should be fixing an api change

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -763,11 +763,11 @@ export class ULabel {
...
         return {
-            "container_id": arguments[0],  // Required
-            "image_data": arguments[1],    // Required
-            "username": arguments[2],      // Required
-            "submit_butons": arguments[3], // Required
-            "subtasks": arguments[4],      // Required
+            "container_id": arguments[0],   // Required
+            "image_data": arguments[1],     // Required
+            "username": arguments[2],       // Required
+            "submit_buttons": arguments[3], // Required
```
Comment (TrevorBurgoyne) [inline src/index.js:769]: since this wasn't a argument previously, does it even make sense to expect it along with the other deprecated args?

**Review by TrevorBurgoyne — COMMENTED** [review]
Note to update the api_spec as it currently doesn't make mention of the `submit_buttons` arg

File: src/index.js (hunk 1 above)
Comment (csolbs24) [reply src/index.js:769]: The `on_submit` argument is the `submit_buttons` argument. In the past only one submit button was supported, but when we expanded support for multiple submit buttons (e.g. a submit and a reject button) everything internally was renamed from `on_submit` to `submit_buttons`. Looks like the api_spec was never updated. Nice catch 👍

Comment (TrevorBurgoyne) [reply src/index.js:769]: oh i see, that makes a lot of sense actually 🤯

File: api_spec.md [hunk 2]
```diff
@@ -61,11 +61,19 @@ In the case of a multi-frame annotation job, an array of URLs may be given. Note
 
 *string* -- This is intended to be a unique ID for the user performing annotations. It will be assigned to each annotation that this user creates during the session.
 
-### `on_submit`
+### `submit_buttons`
```
Comment (TrevorBurgoyne) [inline api_spec.md:64]: higher up in this file (~line 35), the `on_submit` field in the constructor should be changed to `submit_buttons` and the typehints should be updated

**Review by TrevorBurgoyne — APPROVED** [review]
LGTM 👍

**Pull request merged**